throw-if-servlet5/pom.xml

```diff
@@ -18,5 +18,18 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestFile>src/main/resources/META-INF/MANIFEST.MF</manifestFile>
```

Comment:
What about using `<manifestEntries>` instead of `<manifestFile>`, to avoid adding another file to the project?

```
<manifestEntries>
    <Automatic-Module-Name>throwifservlet5</Automatic-Module-Name>
</manifestEntries>
```